mlir/lib/Dialect/Vector/Transforms/VectorEmulateNarrowType.cpp

```diff
@@ -129,9 +128,17 @@ static FailureOr<Operation *> getCompressedMaskOp(OpBuilder &rewriter,
   return newMask;
 }
 
+/// A wrapper function for emitting `vector.extract_strided_slice`. The vector
+/// has to be of 1-D shape.
```

Comment:
[nit] Rather than saying that the vector has to be 1-D, why not say "Extracts 1-D subvector from a 1-D vector". This way, the intent becomes clearer and the requirements are implied ;-)

Just so that you don't have to guess what I had in mind:

> /// A wrapper function to extract a 1-D subvector from the 1-D source vector.

Also, could you remind my why use `vector.extract_strided_slice` rather than `vector.extract`? 

Btw, none of this is a blocker for this PR. These are nice-to-have improvements, thanks!

Reply:
if I understand correctly, `vector.extract` can extract a single element or the whole innermost dimension, but if we want to operate on a certain part of the inner most dimension then we will have to use `vector.extract_strided_slice`? 

Too bad the docs are not super formal so this is how I read it.